packages/test-utils/src/index.js

```diff
@@ -13,7 +13,8 @@ export function hashForEach(hash, func) {
 }
 
 export function getTestDir(dir) {
-  const testRE = /\/[^/]+\.test\.js($|\?.*)/;
+  // Legacy webpack support. I do not know the exact purpose of this file
```

Comment:
yeah I have no clue either lol. We'll probably need to revisit when moving away from karma